src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/NewAzureRmRecoveryServicesVault.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.RecoveryServices.Properties;
+using Microsoft.Azure.Management.RecoveryServices.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+
+namespace Microsoft.Azure.Commands.RecoveryServices
+{
+    /// <summary>
+    /// Used to initiate a vault create operation.
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmRecoveryServicesVault", SupportsShouldProcess = true),
+        OutputType(typeof(ARSVault))]
+    public class NewAzureRmRecoveryServicesVault : RecoveryServicesCmdletBase
```

Comment:
@viverm was this cmdlet intended to go into this PR? This is for a separate module

Reply:
Removing the changes . Not intended .

Reply:
@viverm this file is still here